.github/workflows/slash-commands.yml

```
@@ -0,0 +1,34 @@
+jobs:
+  comment-created:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up GitHub token
+        id: generate-token
+        run: echo "token=${{ github.token }}" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v4
+      - uses: dagger/dagger-for-github@8.0.0
+      - name: slash agent
+        if: github.event.issue.pull_request && contains(github.event.comment.body, '/agent')
+        run: dagger call pull-request-feedback --github-token GH_TOKEN --issue-id ${{ github.event.issue.number }} --feedback "${{ github.event.comment.body }}"
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)